components/lib/atoms/index.js

```diff
@@ -1,20 +1,20 @@
-// import { atom } from "recoil";
+import { atom } from "recoil";
 
-// // ----------- atom with one value here ------------------------
+// ----------- atom with one value here ------------------------
 
-// const openMenuAtom = atom({
-//   key: "openMenuAtom",
-//   default: false,
-// });
+const openMenuAtom = atom({
+  key: "openMenuAtom",
+  default: false,
+});
 
-// const leftNavPinedAtom = atom({
-//   key: "leftNavPinedAtom",
-//   default: false,
-// });
+const leftNavPinedAtom = atom({
+  key: "leftNavPinedAtom",
+  default: false,
+});
 
-// const displaySettingsModalAtom = atom({
-//   key: "displaysettingsModalAtom",
-//   default: false,
-// });
+const displaySettingsModalAtom = atom({
+  key: "displaysettingsModalAtom",
+  default: false,
+});
 
-// export { openMenuAtom, leftNavPinedAtom, displaySettingsModalAtom };
+export { openMenuAtom, leftNavPinedAtom, displaySettingsModalAtom };
```

Comment:
jsx components should be commented before creating the PR

Reply:
i have to use this file for the NavBar component